apps/asap-server/src/controllers/users.ts

```diff
@@ -176,4 +158,41 @@ export default class Users {
 
     return transform(user);
   }
+
+  async sendWelcomeEmail(user: CMSUser) {
+    const code = `ASAP-${uuidV4()}`;
```

Comment:
Does this changes the URL sent to the user? I would prefer to keep the link as `/welcome/${uuid}`.
We can keep the prefix but just for internal use. Similar to what Auth0 does with `google-oauth2|${id}`, `autg0|${id}`.

Reply:
Didn't thought about that. Let me fix it 👍🏽